### PR TITLE
Feat/sidebar pages

### PR DESCRIPTION
**Pull request by JemalSmb**
Added content to each of the sidebar pages

### CODE REVIEW

File: public/ChannelsPreview/ChannelsV.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const ChannelsToC = [
+    {
+        name: 'Mobile Notification Channel',
+        Path: '/ChannelsPreview/mobileNotif.mp4',
+        description: 'Advertisers can post ads that will instantly send mobile notifications to all users.',
```
Comment (shnartho) [inline public/ChannelsPreview/ChannelsV.js:5]:
```suggestion
        description: 'Post ads that will instantly send mobile notifications to all users.',
```

File: public/ChannelsPreview/ChannelsV.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        action: 'Post Ad',
+        live: true
+    },
+    {
+        name: 'Whatsapp Notification Channel',
+        Path: '/ChannelsPreview/whatsappMessage.mp4',
+        description: 'Advertisers can post ads that will instantly send whatsapp notifications to all users.',
```
Comment (shnartho) [inline public/ChannelsPreview/ChannelsV.js:12]:
```suggestion
        description: 'Post ads that will instantly send whatsapp notifications to all users.',
```

File: public/ChannelsPreview/ChannelsV.js (hunk 1 above)
Comment (JemalSmb) [reply public/ChannelsPreview/ChannelsV.js:5]: Straight info is better, I agree

File: public/ChannelsPreview/ChannelsV.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        action: 'Post Ad',
+        live: true
+    },    
+    {
+        name: 'Telegram Notification Channel',
+        Path: '/ChannelsPreview/telegramUsage.jpg',
+        description: 'Advertisers can post ads that will instantly send telegram notifications to all users.',
```
Comment (shnartho) [inline public/ChannelsPreview/ChannelsV.js:19]:
```suggestion
        description: 'Post ads that will instantly send telegram notifications to all users.',
```

File: public/ChannelsPreview/ChannelsV.js (hunk 2 above)
Comment (JemalSmb) [reply public/ChannelsPreview/ChannelsV.js:12]: Makes sense

**Review by shnartho — APPROVED** [review]

**Pull request merged**